src/styles/globals.css

```diff
@@ -54,6 +54,19 @@ body {
     color: transparent;
   }
 
+  /* To style button/container without using :before and :after to have color transition */
+  .brand-bg-gradient-1 {
+    background: linear-gradient(#000000, #000000) padding-box,
+      linear-gradient(
+          103.45deg,
+          #ff12af 22.86%,
+          #0821bb 74.53%,
+          #42f9c2 105.23%
+        )
+        border-box;
+    transition: all 300ms ease-in;
```

Comment:
let's add this separately when using this gradient, since it does not "belong" to `brand-bg-gradient-1`